docs/storing-querying/loading-process/index.md

```diff
@@ -25,11 +26,14 @@ We load data into Redshift using the [RDB Loader](/docs/pipeline-components-and-
 
   </TabItem>
   <TabItem value="bigquery" label="BigQuery">
-
-We load data into BigQuery using the [BigQuery Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/index.md).
-
-<BigQueryLoaderDiagram/>
-
+    <Tabs groupId="bqgroup" queryString lazy>
+        <TabItem value="v2" label="2.x" default>
```

Comment:
Hmmm.... I'd probably go for "Version 2.x" instead of "2.x".  To make it more clear what the button refers to.